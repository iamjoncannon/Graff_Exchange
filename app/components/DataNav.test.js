




describe("DataNav", ()=>{


    it('1) renders successfully', ()=>{

        // let tree = component.toJSON();
        
        // expect(tree).toMatchSnapshot();
    })

    it('renders "dava-nav" div containing spans for all items in data prop', ()=>{})
    it('if item matches selectedDataNavItem, className is "selected"', ()=>{})

})

