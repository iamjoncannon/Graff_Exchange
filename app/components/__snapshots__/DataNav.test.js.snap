// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataNav 1) renders successfully 1`] = `
<div
  className="data-nav"
>
  <span
    onClick={[Function]}
  >
    1
  </span>
  <span
    onClick={[Function]}
  >
    2
  </span>
  <span
    className="selected"
    onClick={[Function]}
  >
    3
  </span>
  <span
    onClick={[Function]}
  >
    5
  </span>
</div>
`;
